app/services/doc_auth/error_generator.rb

```diff
@@ -51,91 +51,103 @@ def initialize(config)
 
     def generate_doc_auth_errors(response_info)
       liveness_enabled = response_info[:liveness_enabled]
-      alert_error_count = response_info[:alert_failure_count]
+      alert_error_count = response_info[:doc_auth_result] == 'Passed' ?
+        0 : response_info[:alert_failure_count]
```

Comment:
Herm...the warn notifier is a good one to think about. This was actually a minor bug with liveness, it wasn't acting quite right when liveness is on and you have no document problems but you have a selfie failure. You can have a passing document that still has a failed alert on it (my license always fails one alert but still passes). This would lead to an improper alert count of two before when there was just a selfie error causing it to be a general error rather than a selfie error. The test that was capturing it was wrong. So when I fixed the test and it started failing that's when I discovered the problem. Now, with liveness turned off this isn't much of a problem but I wanted to fix it in case it ever does get turned on.